modules/setting-up-cpu-manager.adoc

```diff
@@ -9,21 +9,19 @@
 
 .Procedure
 
-. Optional: Label a node:
+. Label a node by running the following command:
 +
 [source,terminal]
 ----
 # oc label node perf-node.example.com cpumanager=true
 ----
 
-. Edit the `MachineConfigPool` of the nodes where CPU Manager should be enabled. In this example, all workers have CPU Manager enabled:
+. To enable CPU Manager for all compute nodes, run the following command and add the `custom-kubelet: cpumanager-enabled` label:
```

Comment:
it should be made very explicit that the label is however just an example and to enable cpumanager it is sufficient the `kubeletconfig` object is applied to the desired set of nodes. IOW, it is not required to neither use this specific label with this specific value. Other than that, it's correct.

Reply:
This looks like an appropriate change to make.

You could also break it apart, i.e.:

> 2. Run the following command: 
> [command]
> In the text editor that opens, add a label that [some criteria].